src/components/HTMLEngineProvider/defaultViewProps/index.native.js

```diff
@@ -0,0 +1,5 @@
+import styles from '../../../styles/styles';
+
+export default {
+    style: [styles.dFlex, styles.userSelectText],
```

Comment:
alignItemsStart was applied to the root renderer for both web and native platforms. While fixing it for web platform I removed it for both platforms and moved it to inner renderers which are responsible for rendering individual components. You can checkout the changes for Image and Attachment renders. This style is needed for both native and web platforms but applying it at root node has unintended effects for web.

Reply:
So the change was needed only for web platforms right, and switch back only for native should make no harm? I came across the problem with `dFlex` on native, as with it applied on root render component, rendered text component span on [whole line](https://github.com/Expensify/App/pull/18495#discussion_r1187302041), and applying it to the renderer itself does nothing as it is intended to be inline rather than block